pkg/ttrpcutil/client_windows.go

```diff
@@ -1,59 +0,0 @@
-// +build windows
-
-/*
-   Copyright The containerd Authors.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-*/
-
-package ttrpcutil
-
-import (
-	"context"
-	"net"
-	"os"
-	"time"
-
-	winio "github.com/Microsoft/go-winio"
-	"github.com/pkg/errors"
-)
-
-func ttrpcDial(address string, timeout time.Duration) (net.Conn, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
-
-	// If there is nobody serving the pipe we limit the timeout for this case to
-	// 5 seconds because any shim that would serve this endpoint should serve it
-	// within 5 seconds.
-	serveTimer := time.NewTimer(5 * time.Second)
```

Comment:
@estesp - And to your point, this one is already odd. This is the ttrpc connection from a shim BACK to containerd.exe for the events publisher ttrpc API. It makes little to no sense to not fail that call immediately if it didn't connect. IE: a shim tried to connect to containerd.exe but couldn't? containerd.exe must not be running.